spec/classroom_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'spec_helper'
+
+describe Classroom do
+  before :each do
+    @classroom = Classroom.new 'label'
+  end
+
+  it 'classroom is an instance of Classroom class' do
+    expect(@classroom).to be_an_instance_of Classroom
+  end
+
+  it 'Should return classroom label' do
+    expect(@classroom.label).to eq 'label'
+  end
```

Comment:
- [ ] I would recommend adding test for all the public method you have for the respective class. 

Hint: 
- Add test for `add_student` method.